src/troubleshooting/backend.rst

```diff
@@ -1,12 +1,626 @@
 Troubleshooting Backend
 =======================
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac ligula et felis aliquam posuere. Etiam ac augue vel tortor porttitor placerat vitae ac libero. Nunc a efficitur lectus. Aliquam malesuada congue arcu quis luctus. Quisque pretium dolor orci, ut ullamcorper felis tincidunt eget. Duis auctor felis ut maximus facilisis. Nam eu feugiat tortor.
+A ``backend``, often also called ``data provider``, is an SSSD child process that
+manages and ``creates the cache``. This process talks to LDAP server, performs
```

Comment:
a cache of what?

Reply:
It was copy paste from original documentation. I trimmed it a little so it should be more straight forward now.

Reply:
Can you please take a look if in the latest version it is more clear?

Reply:
Ack.

Reply:
@justin-stephenson Do you agree with the change as well?

Reply:
s/Cache is a local database/The SSSD Cache is a local database/

Otherwise Ack from me

Reply:
Fixed